code/globalincs/vmallocator.h

```diff
@@ -25,6 +25,16 @@ bool SCP_vector_contains(SCP_vector<T>& vector, T item) {
 	return std::find(vector.begin(), vector.end(), item) != vector.end();
 }
 
+template <typename T>
+bool SCP_vector_inbounds(SCP_vector<T>& vector, int idx) {
+	return ((idx >= 0) && (idx < static_cast<int>(vector.size())));
+}
+
+template <typename T>
+bool SCP_vector_inbounds(SCP_vector<T>& vector, size_t idx) {
+	return ((idx >= 0) && (idx < vector.size()));
```

Comment:
```suggestion
	return idx < vector.size();
```

For convenience.  Feel free to ignore if you have already made changes or want to alter the formatting.